### PR TITLE
solution

### PR DESCRIPTION
**Pull request by ihorandrianov**
[DEMO LINK](https://ihorandrianov.github.io/react_static-list-of-todos/)

### CODE REVIEW

File: src/components/TodoInfo/TodoInfo.tsx [hunk 1]
```diff
@@ -1,4 +1,27 @@
...
+import { Todo } from '../../types/Todo';
+
+type TodoProp = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<TodoProp> = ({ todo }) => {
+  const {
+    id, userId, title, completed, user,
+  } = todo;
```
Comment (MaxManis) [inline src/components/TodoInfo/TodoInfo.tsx:13]:
```suggestion
  const {
    id,
    userId,
    title,
    completed,
    user,
  } = todo;
```

File: src/components/TodoList/TodoList.tsx [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type TodosProps = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<TodosProps> = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map((todo) => (<TodoInfo todo={todo} />))}
```
Comment (MaxManis) [inline src/components/TodoList/TodoList.tsx:12]:
```suggestion
    {todos.map((todo) => (
        <TodoInfo todo={todo} />
    ))}
```
))

**Review by MaxManis — APPROVED** [review]
Good job!)

**Review by OleksiiBlezniuk — CHANGES_REQUESTED** [review]

**Review by silchyn — CHANGES_REQUESTED** [review]
> Use semantic tags for list and it's items

File: src/components/TodoInfo/TodoInfo.tsx [hunk 3]
```diff
@@ -1,4 +1,29 @@
...
+};
+
+export const TodoInfo: React.FC<TodoProp> = ({ todo }) => {
+  const {
+    userId, title, completed, user,
+  } = todo;
+
+  return (
+    <li
+      className={classNames('TodoInfo', { 'TodoInfo--completed': completed })}
```
Comment (naziamoff) [inline src/components/TodoInfo/TodoInfo.tsx:18]: It is better to keep it maximally clear to those who read
```suggestion
      className={classNames('TodoInfo', { 
        'TodoInfo--completed': completed 
      })}
```

File: src/components/TodoList/TodoList.tsx [hunk 4]
```diff
@@ -1,4 +1,14 @@
...
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type TodosProps = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<TodosProps> = ({ todos }) => (
+  <ul className="TodoList">
+    {todos.map((todo) => (<TodoInfo todo={todo} key={todo.id} />))}
```
Comment (naziamoff) [inline src/components/TodoList/TodoList.tsx:12]: I understand the temptations to provide one-line solution 😄 
But clearness of the code is our top priority
```suggestion
    {todos.map((todo) => (
      <TodoInfo todo={todo} key={todo.id} />
    ))}
```

**Review by naziamoff — APPROVED** [review]
Good job! There are some comments about formatting though

**Pull request closed without merging**